### DISCUSSION TITLE
deterministic-testing-in-go/

### DISCUSSION THREAD

**Discussion by giscus** [JungWinter/blog#47] [Comments]
# deterministic-testing-in-go/

지난 7월 Go To Jeju 2024 행사와 10월의 GopherCon Korea 2024 행사에서 《Deterministic testing in Go》 주제로 발표를 했다.
발표에서는 코드에서 time.Now()를 사용하더라도 어떻게 시간에 구애…

https://www.winterjung.dev/deterministic-testing-in-go/

**Comment by seongwoopark** [comment]
최근 go 테스트코드에서 귀찮았던 내용인데 감사합니다~
